pkg/cmd/build/new.go

```diff
@@ -17,6 +19,7 @@ func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
 	var commit string
 	var message string
 	var pipeline string
+	var confirm bool
```

Comment:
Add a cmd flag around this as well. Then you can use the WithHiddenFunc of huh forms to optionally show the form if its not set already?

Reply:
Using --confirm will now skip the input.